### ISSUE TITLE
tests to shared?

### ISSUE THREAD

**Issue by tofutim** [praeclarum/sqlite-net#487]
How do you feel about moving the test cs files to a Shared Project instead of using links to files for iOS? I wanted to test a couple different bundles from SQLitePCL.raw and that would make things a bit smoother. The sqlite3.dll test would become:

![image](https://cloud.githubusercontent.com/assets/157766/21287478/5d35e622-c421-11e6-9b30-f84444c86d99.png)

and I would make a new test, such as SQLite3.Tests.bundle_green, etc.

**Comment by praeclarum** [comment]
Yeah this would enable me to run different test scenarios too - specifically a reference to raw test.
